README.md

```diff
@@ -2,6 +2,38 @@
 
 A pilot project to extract medical indications (when a compound treats a disease) from the literature using [Snorkel](https://github.com/HazyResearch/snorkel).
 
+##Installing/Setting Up The Conda Environment
+Snorkeling uses [conda](http://conda.pydata.org/docs/intro.html) as a python package manager. Before moving on to the instructions below, please make sure to have it installed. [Download conda here!!](https://www.continuum.io/downloads) Also, make sure to clone the snorkel repository found [here](https://github.com/HazyResearch/snorkel).  
+  
+Once everything has been installed and cloned, type following command in the terminal: 
+
+```bash
+conda env create --file environment.yml
+``` 
+
+After everything has been isntalled, activate  the environment by using the following command: 
+
+```bash
+source activate snorkeling
+```  
+
+_Note_: If you want to leave the environment enter the following command:
+
+```bash
+source deactivate 
+```
+
+##Running The Project
+After modifying the environment script please type in the following command: 
+
+```bash 
+bash run.sh
+```  
+
+_Note_: Before running the script above please edit the following script [set_env.sh](set_env.sh). Change the `SNORKELHOME` variable to point to the snorkel directory you cloned above. Depending on where this repository is located you may have to change the `WORKINGPATH` variable. If you have to change it, make sure it points to this repository.
```

Comment:
Is there a reason why snorkel must be installed using this method rather than just using pip? Ideally, we could free ourselves from having to deal with these paths.